content/blog/2016-09-28-interview-with-incopro.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Interview with Incopro
+created_at: 2016-09-28
+kind: article
+author_name: Brian Brazil
+---
+
+*Continuing our series of interviews with users of Prometheus, INCOPRO talks
+about how they went from Zabbix and Monit to Prometheus.*
+
+## Can you tell us about yourself and what Incorpo does?
```

Comment:
Fixed. Looking at their website, their branding is the usual capitalization so changed.